dbusmock/testcase.py

```diff
@@ -28,6 +30,260 @@
 
 from dbusmock.mockobject import MOCK_IFACE, OBJECT_MANAGER_IFACE, load_module
 
+logger = logging.getLogger("dbusmock")
+logger.setLevel(logging.DEBUG)
+
+
+class BusType(enum.Enum):
+    SESSION = "session"
+    SYSTEM = "system"
+
+    @staticmethod
+    def from_bool(system_bus: bool) -> "BusType":
```

Comment:
Why the quoted return type? Doesn't c8s's python 3.6 understand an unquoted class name yet?

Reply:
The type isn't available until it's fully defined and we're inside `BusType` here, so it's not yet defined at this point. But after a bit of googling this should be `-> Self` instead.

Reply:
Turns out `Self` is not available until [python 3.11](https://peps.python.org/pep-0673/), so I'd say let's keep the quoted one.

Reply:
Ah, ok. We can clean these up once RHEL 8 goes into maintenance mode.